src/components/ListsTable/ListsTable.tsx

```diff
@@ -18,7 +18,7 @@ export interface ListsTableProps {
 
 export const ListsTable: FC<ListsTableProps> = ({
   activeFilters,
-  setTotalRecords = () => {}
+  setTotalRecords = noop
 }) => {
```

Comment:
Could you please move functionality connected with polling from this component to the parent? It will be necessary in any case in the future.

Reply:
I'm confused, I don't believe there is any functionality related to polling in this component. Are you thinking of `ListInformationPage.tsx`?